shared-svcs-stamp.json

```diff
@@ -0,0 +1,287 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "0.0.0.5",
+    "parameters": {
+        "location": {
```

Comment:
add brazil as an option.

Reply:
Done | Addressed from [9ce2c](https://github.com/mspnp/aks-secure-baseline/pull/158/commits/9ce2c466ce890f57b6985214e9f7a8dbe8fe6fa3)

cc/ @ckittel  I think we included [all products](https://azure.microsoft.com/en-us/global-infrastructure/services/?regions=brazil-south,non-regional,brazil-southeast,us-east-2,us-central&products=storage,azure-bastion,frontdoor,azure-firewall,virtual-network,load-balancer,application-gateway,key-vault,active-directory,monitor,container-registry,kubernetes-service)